src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -223,7 +223,7 @@ nuget install ninject -o c:\foo</value>
     <value>List all versions of a package. By default, only the latest package version is displayed.</value>
   </data>
   <data name="ListCommandDescription" xml:space="preserve">
-    <value>Displays a list of packages from a given source. If no sources are specified, all sources defined in %AppData%\NuGet\NuGet.config are used. If NuGet.config specifies no sources, uses the default NuGet feed.</value>
+    <value>(Deprecated) Displays a list of packages from a given source. If no sources are specified, all sources defined in %AppData%\NuGet\NuGet.config are used. If NuGet.config specifies no sources, uses the default NuGet feed.</value>
```

Comment:
Should this be more prominent? 

Special chars/new line/ all caps? 

@anangaur

Reply:
Now it is all caps. But, the message is generated dinamically

Reply:
What about deprecated colored word?

Reply:
We have not used colored words much beyond yellow for warning and red for error. 

Personally I think caps is fine.